brave-opentracing/src/main/java/io/opentracing/BraveTracerImpl.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2016 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing;
+
+import java.util.Map;
+
+import com.github.kristofa.brave.Brave;
+import com.github.kristofa.brave.IdConversion;
+import com.github.kristofa.brave.SpanId;
+import com.github.kristofa.brave.http.BraveHttpHeaders;
+
+import io.opentracing.propagation.Format;
+import java.util.HashMap;
+
+public final class BraveTracerImpl extends AbstractTracer {
```

Comment:
Since there's no BraveTracer interface, I think it would be nicer to name this class BraveTracer instead of BraveTracerImpl. Similarly with other types in this PR. We don't tend to follow the *Impl naming convention here unless there's actually a name conflict.